docs/compliance/cis/docker_ce.md

```diff
@@ -0,0 +1,12 @@
+---
+description: CIS Docker CE Benchmark information
+keywords: standards, compliance, security, benchmark
+title: CIS Docker CE benchmark
+---
+
+The Center for Internet Security (CIS) Docker Community Edition (CE) Benchmark is a reference document that can be used by system administrators, security and audit professionals and other IT roles in order to establish a secure configuration baseline for the Docker CE Engine. CIS Benchmarks are developed by an open community of security practitioners and licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 Internal Public License](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode).
+
+The latest benchmark for Docker CE 17.06 (CIS Benchmark v1.1.0) can be found [here](https://www.cisecurity.org/benchmark/docker/). Below is a list of tools and resources that can be used to automate the validation of an instance of Docker CE Engine against the CIS Docker CE Benchmark:
```

Comment:
CE 17.06 is no longer supported, so I am not sure about this. Remember, CE releases are only valid for one month after the next CE Stable comes out (so support for 17.06 would have ended some time in October).

Reply:
Yea, we'll need to denote this then. I can make a note that the CIS Benchmarks are updated against the Stable CE releases and *only* when there are significant changes that impact the nature of the hardening guidance in the benchmark. And that this cadence could lag behind the supported lifetime of a particular CE release ... thoughts?

Reply:
I'm not the one who should make this call. Someone in Product or Marketing, I imagine. Maybe take this discussion offline with them.

Reply:
Just pushed another commit that makes this a bit more clear